src/commcare_cloud/commands/inventory_lookup/inventory_lookup.py

```diff
@@ -269,3 +291,74 @@ def _get_ssh_args(remote_command):
         else:
             ssh_args = _get_ssh_args(remote_command)
             return Ssh(self.parser).run(args, ssh_args)
+
+
+class ForwardPort(CommandBase):
+    command = 'forward-port'
+    help = """
+    Port forward to access a remote admin console
+    """
+    _SERVICES = {
+        'flower': ('celery[0]', 5555),
+        'couch': ('couchdb2_proxy[0]', 25984),
+        'elasticsearch': ('elasticsearch[0]', 9200),
+    }
+
+    arguments = (
+        Argument('service', choices=_SERVICES.keys(), help=f"""
+            The remote service to port forward. Must be one of {','.join(sorted(_SERVICES.keys()))}.
+        """),
+    )
+
+    def run(self, args, unknown_args):
+        environment = get_environment(args.env_name)
+        nice_name = environment.terraform_config.account_alias
+        remote_host_key, remote_port = self._SERVICES[args.service]
+        loopback_address = f'127.0.{environment.terraform_config.vpc_begin_range}'
+        remote_host = lookup_server_address(args.env_name, remote_host_key)
+
+        while not self.is_loopback_address_set_up(loopback_address):
+            puts(color_error('To make this work you will need to run set up a special loopback address on your local machine:'))
+            puts(color_notice(f'  - Mac: Run `sudo ifconfig lo0 alias {loopback_address}`.'))
+            puts(color_notice(f'  - Linux: Run `sudo ip addr add {loopback_address}/8 dev lo`.'))
```

Comment:
Agreed this is really dangerous and makes the current implementation a non-starter. Specifically that someone could go to http://commcare-staging:5984/ and see local data when they expect to see remote data opens the door for exactly the type of confusion we want to avoid.

Reply:
This is something we can easily fix by binding for example on port 0, obtaining a random available port.
I think it's still a good idea to maintain the current implementation in place, because we want to have the environment name spelled out in the link, clearly visible in the open window/tab/terminal. We just need to work around these potential conflicts, and the above should be an effective solution.